database/migrations/2020_01_13_000001_create_order_products_table.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateOrderProductsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('order_product', function (Blueprint $table) {
+            $table->bigIncrements('id');
+            $table->unsignedBigInteger('order_id')->index();
+            $table->foreign('order_id')->references('id')->on('orders');
+            $table->unsignedBigInteger('product_id')->unsigned()->index();
+            $table->foreign('product_id')->references('id')->on('products');
```

Comment:
I personally prefer this style just due to readability:

```php
$table->unsignedBigInteger('order_id')->index();
$table->unsignedBigInteger('product_id')->unsigned()->index();

$table->foreign('order_id')->references('id')->on('orders');
$table->foreign('product_id')->references('id')->on('products');
```

Reply:
I agree that it's easier to read and will make that change.